app/views/events/_form.html.erb

```diff
@@ -31,22 +31,9 @@
   <div class="form-group">
     <%= f.label :description, :class => 'control-label col-lg-2' %>
     <div class="col-lg-10">
-      <%= f.text_area :description, :class => 'form-control', :id => 'description',  :placeholder => '', :rows => 20 %>
+      <%= f.text_area :description, :class => 'form-control', :id => 'description',  :placeholder => '', :rows => 20, :spellcheck => "false" %>
       <script>
-        var placeholder = '## Untertitel \nLorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.\n \n ## Lernziele\n* a\n* b\n* c\n\n## Hinweise\n* a\n* b\n* c';
-        $("#description").attr('placeholder', placeholder);
-
-        $("#description").focus(function(){
-            if($(this).val() === placeholder){
-                $(this).attr('placeholder', '');
-            }
-        });
-
-        $("#description").blur(function(){
-            if($(this).val() ===''){
-                $(this).attr('placeholder', placeholder);
-            }    
-        });
+        var PLACEHOLDER = '<%= I18n.t "events.description_placeholder" %>';
```

Comment:
Maybe prefix the constant name as `EVENT_DESCRIPTION_PLACEHOLDER`, just in case anyone else was to introduce another placeholder somewhere.